packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -1191,119 +1354,135 @@ function CheckoutLineItem( {
 				</MobileGiftWrapper>
 			) }
 			<LineItemTitle id={ itemSpanId } isSummary={ isSummary }>
-				{ label }
+				{ hasCheckoutVersion2 && isRenewal ? `${ label } Renewal` : label }
```

Comment:
While not in the Figma, I felt this was a good place to list renewal text, it also cuts down on a heap of code found in checkout-labels.ts:

![image](https://github.com/Automattic/wp-calypso/assets/16580129/517f78a4-8920-42d5-951a-2ebbd6b7a994)